### PR TITLE
[13.0][MIG] stock_request_picking_type

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard migration.

Waiting for
- [x] [13.0][MIG] stock_request_submit https://github.com/OCA/stock-logistics-warehouse/pull/866

### CODE REVIEW

**Comment by dreispt** [comment]
Build is red. Is this a WIP?

**Comment by dreispt** [comment]
Got my answer - this depends on #866

**Review by bodedra — CHANGES_REQUESTED** [review]
@ps-tubtim 

It fails when I click on Inventory menu.

Screenshot from 2020-10-21 15-08-36

**Comment by dreispt** [comment]
🚨 The `stock_request_purchase` tests are failing, because of a `picking_type_id` required field on `stock_request_order`.

File: stock_request_picking_type/models/stock_picking_type.py
Comment (RLeeOSI) [inline stock_request_picking_type/models/stock_picking_type.py:17]: Compute method needs explicit values for all records

Comment (ps-tubtim) [reply stock_request_picking_type/models/stock_picking_type.py:17]: @RLeeOSI Thank you for your review. Edited.

**Comment by bodedra** [comment]
@ps-tubtim Would you please attend @dreispt [comment](https://github.com/OCA/stock-logistics-warehouse/pull/867#issuecomment-714783943)?

Screenshot from 2020-10-29 08-23-28

**Comment by RLeeOSI** [comment]
@bodedra This is a multicompany issue. On install, a stock.picking.type is created with type 'stock_request_order' . The stock.request has a default method to find a stock.picking.type with type 'stock_request_order' with warehouse.company_id matching user's company_id or company in context. If the record created on install is for Company 1, then any stock request order created in Company 2 will fail because there's no stock.picking.type found. 

What's the best way to handle this? A stock.picking.type needs to be created in each company. We can make that part of the documentation. To make it clearer, we could make the field visible with a domain filter type = 'stock_request_order' so that the user has some idea of what's missing. Your opinion?

**Comment by bodedra** [comment]
@RLeeOSI I agree with your idea.

**Comment by RLeeOSI** [comment]
@ps-tubtim I have a [PR on your fork](https://github.com/ecosoft-odoo/stock-logistics-warehouse/pull/1) to fix issue raised above

**Comment by ps-tubtim** [comment]
@All Sorry, I make a mistake. Move to https://github.com/OCA/stock-logistics-warehouse/pull/1010

**Comment by rousseldenis** [comment]
@ps-tubtim In OCA now, we don't use anymore the Github Merge button. Please use ocabot for that purpose.

https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92

There are many operations done before the real merge done by the bot.

Thanks!

**Pull request merged**
